.github/workflows/ci.yml

```diff
@@ -8,19 +8,24 @@ on:
   release:
     types: [published]
 
-concurrency:
-  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
-  cancel-in-progress: true
-
 env:
   CARGO_TERM_COLOR: always
   DASEL_VERSION: https://github.com/TomWright/dasel/releases/download/v1.24.3/dasel_linux_amd64
   RUSTFLAGS: "-D warnings"
   RUST_VERSION: 1.61.0
-  FORC_VERSION: 0.16.0
+  FORC_VERSION: 0.17.0
 
 jobs:
+  cancel-previous-runs:
```

Comment:
Why are you reverting #445?

Reply:
For some reason it did not merge CI from the master. :thinking: